packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {
+    OwnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { Semver } from "../universal/Semver.sol";
+
+/**
+ * @title SystemConfig
```

Comment:
I'm ok with a follow up PR that adds devdocs and unit tests for this contract, its relatively straight forward and the e2e tests technically do cover the functionality here, but that isn't an excuse for not having unit tests

Reply:
Yep, need some help with contract testing, although not urgent (it'll will be a while until we can land this breaking change). Maybe you or @maurelian have time end this cycle, or start next cycle?

Reply:
ENG-2949 is tracking this now